android/anc/src/main/java/org/smartregister/fhircore/anc/ui/details/bmicompute/BmiQuestionnaireViewModel.kt

```diff
@@ -58,6 +58,22 @@ class BmiQuestionnaireViewModel @Inject constructor(val patientRepository: Patie
     const val BMI_CATEGORY_UNDERWEIGHT_MAX_THRESHOLD = 18.5
     const val BMI_CATEGORY_NORMAL_MAX_THRESHOLD = 25
     const val BMI_CATEGORY_OVERWEIGHT_MAX_THRESHOLD = 30
+
+    fun getHeightAsPerMetricUnit(inputHeight: Double, unitModeMetric: Boolean): Double {
+      return if (unitModeMetric) {
+        inputHeight
+      } else {
+        inputHeight / HEIGHT_INCH_METER_MULTIPLIER
+      }
+    }
+
+    fun getWeightAsPerMetricUnit(inputWeight: Double, unitModeMetric: Boolean): Double {
+      return if (unitModeMetric) {
+        inputWeight
+      } else {
+        inputWeight / WEIGHT_POUND_KG_MULTIPLIER
+      }
+    }
   }
```

Comment:
These two functions can be refactored into one. Use a boolean flag/enum constants to differentiate Height/Weight computation

Reply:
done, remove these functions as no more required after refactoring